app/code/Magento/Deploy/Console/DeployStaticOptions.php

```diff
@@ -127,6 +127,11 @@ class DeployStaticOptions
      */
     const NO_LESS = 'no-less';
 
+    /**
+     * Key for not compiling parent themes
+     */
+    const NO_PARENT = 'no-parent';
```

Comment:
This is probably backward incompatible change, but this class insn't marked as `@api` and that's the most logical place to put this new argument name